### ISSUE TITLE
iOS: Changing volume while audio is playing breaks/pauses audio

### ISSUE THREAD

**Issue by IbrahimYildirim** [ryanheise/just_audio#1106]
**iOS Only**
When calling `setVolume()` with value from a `Slider` the playing audio pauses/breaks and continues only after the slider thumb has been relased

[Minimal reproduction project](https://github.com/IbrahimYildirim/JustAudioVolumeChangeError/tree/main)

**To Reproduce - In the MRP:**
1. Allow Microphone Permission
2. Make a 3-4 second recording
3. Click Play
4. Adjust volume while audio is playing (note a 3 sec silent audio has been added to make space between recording)

**Expected behavior**
Volume change happening without audio disturbance

**Smartphone:**
 - Device: iPhone 12 Pro
 - OS: 16.6.1

**Flutter SDK version**
```
[✓] Flutter (Channel stable, 3.13.8, on macOS 13.2 22D49 darwin-arm64, locale en-DK)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
[✓] Xcode - develop for iOS and macOS (Xcode 14.3.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2022.1)
[✓] VS Code (version 1.75.1)
[✓] Connected device (3 available)
[✓] Network resources
```

**Additional context**
This started to happen recently on my affirmation app where users record affirmations with their own voice and have it played in loop with a short break in between. This used to be smooth but now it's lagging when changing volume

**Comment by IbrahimYildirim** [comment]
@ryanheise I accidentally adjusted the headings and the bot didn't like that. Switched it back to original now, but it's still marked as **closed** 

Please let me know if you need anything or want me to open a new issue regarding this and i'll do it - I just didn't want to make a duplicate before asking.
